helm_deploy/laa-estimate-eligibility/templates/metabase/metabase-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.metabase.enabled }}
+{{- $fullName := include "app.fullname" . -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}-metabase
+spec:
+  replicas: 1 # Metabase does not support horizontal scaling.
+  selector:
+    matchLabels:
+      app: {{ $fullName }}-metabase
+  template:
+    metadata:
+      labels:
+        app: {{ $fullName }}-metabase
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "laa-estimate-eligibility.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+      - name: metabase
+        image: metabase/metabase:v0.47.8
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+          - containerPort: 3000
+        resources:
+          requests:
+            cpu: 25m
+            memory: 1Gi
+          limits:
+            cpu: 500m
+            memory: 3Gi
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 1000
+          runAsGroup: 1000
+        env:
+          - name: MB_DB_CONNECTION_URI
+            valueFrom:
+              secretKeyRef:
+                name: metabase
+                key: jdbc_url
```

Comment:
So are we creating a new RDS database for Metabase? Or is this URL somehow connected to our existing (staging) database? I'd have thought we'd only want Metabase to talk to our existing database, in which case this MB_DB_CONNECTION_URI maybe needs to be constructed from the `rds-postgresql-instance-output` secrets?

Reply:
you are correct, we should be linking it to our existing RDS, fixed.